package.json

```diff
@@ -44,5 +45,14 @@
         "time-grunt": "^2.0.0",
         "underscore": "^1.9.1",
         "underscore-deep-extend": "^1.1.5"
+    },
+    "devDependencies": {
```

Comment:
Are Adapt Framework node modules installed in production mode in the AAT?

Reply:
They appear to be:
https://github.com/adaptlearning/adapt_authoring/blob/master/lib/installHelpers.js#L666

Reply:
Cool. ta.